### PR TITLE
update to the css styleguide

### PR DESCRIPTION
**Pull request by jasonjeffrey**
Hi Guys, i've made an update to CSS styleguide after the discussions from the previous couple of days. if you could all have a look at make sure we are all happy with these changes and all the guidelines in general. 

I will only merge this once we everyone in the team has LGTMed the Pull request

### CODE REVIEW

File: CSS.md [hunk 1]
```diff
@@ -87,65 +86,48 @@ or the inside page.
 Theme rules are similar to state rules in that they describe how modules or layouts might look. Most sites will just
 use a single default theme.
 
+##CSS Rules
+With the exception of the base rules wherever possible only use classes instead of ID's and elements.
```
Comment (rickyclegg) [inline CSS.md]: Do we ever permit and ID selector on our own CSS, not governed by a framework?

File: CSS.md [hunk 2]
```diff
@@ -87,65 +86,48 @@ or the inside page.
...
+avoid over qualifying class names with extra class and element selectors.
 ```css
 /* Not recommended */
-ul#example {}
+.new.example {}
 div.error {}
 
 /* Recommended */
-#example {}
+.example {}
```
Comment (rickyclegg) [inline CSS.md]: Should we add an example for ```.example-error``` as a recommended form?

File: CSS.md (hunk 1 above)
Comment (chapperz) [inline CSS.md]: I can't think of a reason I would use # in CSS now.

File: CSS.md [hunk 3]
```diff
@@ -171,7 +153,26 @@ Omit unit specification after “0” values.
...
+```css
+/* Not recommended */
+color: rgb(255, 255, 255);
 
+/* Recommended */
+color: #ffffff;
+color: rgba(255, 255, 255, 1);
+```
 ###Hexidecimal Notation
 Use 6 lowercase charators for hexidecimal notation
```
Comment (chapperz) [inline CSS.md]: charators = characters

File: CSS.md [hunk 4]
```diff
@@ -197,6 +196,49 @@ padding-bottom: 0;
...
+
+html {
+  font-family: 'open sans', arial, sans-serif;
+}
+```
+###Use of !important statements
+DONT! - this should only be used as a last resort.
+
+###Property Fallbacks
+property fallbacks should be grouped together at the bottom of each rule as if they were a vendor specific properties. Also add a comment to explain its use.
```
Comment (chapperz) [inline CSS.md]: Not sure I like the comment for the fallback.

File: CSS.md (hunk 1 above)
Comment (jasonjeffrey) [inline CSS.md]: i dont think there is a need to use ID's unless constrained by a framework. I feel ID's could still be used by javascript to target certain elements if we are not using jQuery.

File: CSS.md (hunk 2 above)
Comment (jasonjeffrey) [inline CSS.md]: ive added this to the examples however as error is the item i have flipped the selector name to .error-example

File: CSS.md (hunk 3 above)
Comment (jasonjeffrey) [inline CSS.md]: whoops nice spot

File: CSS.md (hunk 4 above)
Comment (jasonjeffrey) [inline CSS.md]: after further discussion and research on this point. assuming we support IE9+, I would like to suggest changing this rule to the following:

###Property Fallbacks
property fallbacks as seen in the example below should be avoided wherever possible. Vendor specific or filter properties should be used instead.
```css
/* Example */
background: #013157;
background: #013157 linear-gradient(to right, #013157, #7a99ad);
```

**Comment by rickyclegg** [comment]
LGTM

**Pull request merged**